helm/charts/kratos/templates/hpa.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```

nit: In hydra we already use beta2
```suggestion
apiVersion: autoscaling/v2beta2
```